tests/unit/test_battery_types.py

```
@@ -0,0 +1,132 @@
+            "battery_quantity",
+            "is_rechargeable",
+            "typical_life_days",
+        ]
+
+        for model, spec in BATTERY_SPECS.items():
+            for field in required_fields:
+                assert field in spec, f"Model {model} missing field {field}"
+
+    def test_battery_specs_types(self):
+        """Test that battery spec values have correct types."""
+        for model, spec in BATTERY_SPECS.items():
```

Ruff (0.15.12)

[warning] Loop control variable `model` not used within loop body

Rename unused `model` to `_model`

(B007)